shm/src/Cargo.cpp

```diff
@@ -0,0 +1,3 @@
+bool Cargo::operator==(const Cargo& cargo){
+    return cargo.name_ == name_;
+}
```

Comment:
New line at the end of file. Not really needed in modern compilers, but removes this red warning circle here.